.golangci.yml

```diff
@@ -1,3 +1,47 @@
+run:
+  timeout: 5m
 linters:
+  # Enable all available linters.
+  # Default: false
+  enable-all: true
+  enable:
+
   disable:
+    # deprecated lints
+    - ifshort
+    - scopelint
+    # TODO: fix lint erros - ignore until fixed
+    - forcetypeassert
+    - godox
+    - gomnd
+    - gochecknoglobals
+    - containedctx
+    - forbidigo
+    - testpackage
+    - funlen
+    - goconst
+    - godot
+    - gomoddirectives
+    - lll
+    - tagliatelle
+    - nosnakecase
+    - nlreturn
+    - goprintffuncname
+    - usestdlibvars
+    - maintidx
+    - wsl
+    - nestif
+    - gocognit
+    - gofmt
+    - dupl
+    - gocyclo
+    - gochecknoinits
+    - gci
+    - gofumpt
+    - whitespace
+    - prealloc
+    - goimports
+    - nosprintfhostport
+    - gosec
     - errcheck
+  fast: true
```

Comment:
Should we run without this flag?

Reply:
yes, i've revised the docs and updated the file